### DISCUSSION TITLE
docker_teslamate_1 exited with code 1 after upgrade to 1.23.4

### DISCUSSION THREAD

**Discussion by seabie** [teslamate-org/teslamate#1763] [General]
I just ran the update procedure to move to 1.23.4 (I think I was on the previous release but it is possible I was one more behind?).  I'm now seeing the following trying to bring everything up (abbreviated log for what looks like the first error):

`teslamate_1  | 2021-06-28 00:43:39.474 [info] Application telemetry exited: :stopped
teslamate_1  | 2021-06-28 00:43:39.543 [info] Application eex exited: :stopped
teslamate_1  | ** (MatchError) no match of right hand side value: {:error, {:decimal, {{:case_clause, {:ok, []}}, [{:application_controller, :make_appl, 1, [file: 'application_controller.erl', line: 1461]}, {:application_controller, :do_load_application, 2, [file: 'application_controller.erl', line: 1273]}, {:application_controller, :handle_call, 3, [file: 'application_controller.erl', line: 628]}, {:gen_server, :try_handle_call, 4, [file: 'gen_server.erl', line: 721]}, {:gen_server, :handle_msg, 6, [file: 'gen_server.erl', line: 750]}]}}}
teslamate_1  |     (ecto_sql 3.6.2) lib/ecto/migrator.ex:132: anonymous fn/2 in Ecto.Migrator.with_repo/3
teslamate_1  |     (elixir 1.12.1) lib/enum.ex:3865: Enum.flat_map_list/2
teslamate_1  |     (ecto_sql 3.6.2) lib/ecto/migrator.ex:131: Ecto.Migrator.with_repo/3
teslamate_1  |     (teslamate 1.23.4) lib/teslamate/release.ex:9: anonymous fn/2 in TeslaMate.Release.migrate/0
teslamate_1  |     (elixir 1.12.1) lib/enum.ex:2356: Enum."-reduce/3-lists^foldl/2-0-"/3
teslamate_1  |     (teslamate 1.23.4) lib/teslamate/release.ex:8: TeslaMate.Release.migrate/0
teslamate_1  |     (stdlib 3.15.1) erl_eval.erl:685: :erl_eval.do_apply/6
docker_teslamate_1 exited with code 1`

After looking at the latest docker-compose.yml file I noticed that Postgres was updated from v12 (what I was on) to v13.  I executed the backup->Postgres docker-compose change to 13->docker-compose pull->Restore->docker-compose down->docker-compose up sequence and I'm still seeing the same issue.

Any advice?  We just bought our second Tesla recently if that has any bearing on the issue.  Was the first time I went to use Teslamate after the purchase, noticed I was behind a revision, which started this issue.

**Comment by adriankumpf** [comment]
Hey @seabie, can you attach the surrounding logs too? On which OS are you running TeslaMate?

**Reply by seabie** [reply]
@adriankumpf Sure.  I am running on a raspberrypi:

```
pi@raspberrypi:~ $ uname -a
Linux raspberrypi 4.19.97-v7+ #1294 SMP Thu Jan 30 13:15:58 GMT 2020 armv7l GNU/Linux
```

I was about to reply and ask how / where logs exist for Teslamate?  I poked around on the docs page and did not see anything on where to find logs?  I can dump the output of docker logs <container> if that is what would be useful, but I just ran it and it seems like it is full of the error referenced above.  

Thank you for responding (and all the work into this project!!)

**Reply by seabie** [reply]
[teslamate_exit_logs.txt](https://github.com/adriankumpf/teslamate/files/6729734/teslamate_exit_logs.txt)
Found this post https://github.com/adriankumpf/teslamate/discussions/1087.  I attached the logs

**Reply by seabie** [reply]
Are there different logs on the filesystem that are separate from stdout/stderr docker logs?  Just trying to make sure I attached the correct logs
